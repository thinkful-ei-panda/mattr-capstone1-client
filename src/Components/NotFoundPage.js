import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import Nav from './Nav'


export default class NotFoundPage extends Component {
   
    render() {
        return (
            <div>
                 <Nav />  
                <h3>We don't know what you're looking for. <Link to='/Home'>Go Back</Link> </h3>
        
        </div>
        )
    }
}
