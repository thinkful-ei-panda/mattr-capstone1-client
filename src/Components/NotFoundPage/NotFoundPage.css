.go-back {
  font-size: 1.8em;
  text-align: center;
}

.not-found-box {
  text-align: center;
}

.not-found {
  font-size: 1.5em;
  width: 95%;
}

@media screen and (min-width: 1200px) {
  .go-back {
    font-size: 2em;
  }
}
