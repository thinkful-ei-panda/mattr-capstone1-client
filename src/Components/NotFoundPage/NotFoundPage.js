import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import Nav from '../Nav/Nav'
import './NotFoundPage.css'


export default class NotFoundPage extends Component {
   
    render() {
        return (
            <div className='not-found-box'>
                 <Nav />  
                <h3 className='not-found'>We don't know what you're looking for. </h3>
                <Link to='/Home' className='go-back' >Go Back</Link>
        
        </div>
        )
    }
}
