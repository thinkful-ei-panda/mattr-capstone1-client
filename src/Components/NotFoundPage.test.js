import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import NotFoundPage from './NotFoundPage';
describe(`NotFoundPage Component`, () => {
    describe(`Smoke test`, () => {
        it(`Renders without crashing`, () => {
            const div = document.createElement('div');
            ReactDOM.render(
                <BrowserRouter>
                    <NotFoundPage />
                </BrowserRouter>
                , div);
            ReactDOM.unmountComponentAtNode(div);
        });
    });
    
    describe(`Snapshot test`, () => {
        it(`Renders the UI as expected`, () => {
            const tree = renderer
                .create(
                    <BrowserRouter>
                        <NotFoundPage />
                    </BrowserRouter>
                )
                .toJSON()
            expect(tree).toMatchSnapshot()
        });
    });
})