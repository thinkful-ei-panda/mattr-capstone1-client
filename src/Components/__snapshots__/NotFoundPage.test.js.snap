// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component Snapshot test Renders the UI as expected 1`] = `
<div>
  <div
    className="Nav"
  >
    <a
      href="/Home"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/Election"
      onClick={[Function]}
    >
      Elections
    </a>
    <a
      href="/RegistrationForm"
      onClick={[Function]}
    >
       
      Register
       
    </a>
    <a
      href="/Login"
      onClick={[Function]}
    >
      Login
    </a>
  </div>
  <h3>
    We don't know what you're looking for. 
    <a
      href="/Home"
      onClick={[Function]}
    >
      Go Back
    </a>
     
  </h3>
</div>
`;
